doc/classes/Quaternion.xml

```diff
@@ -39,7 +39,8 @@
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Constructs a quaternion that will rotate around the given axis by the specified angle. The axis must be a normalized vector.
+				Constructs a quaternion that will rotate around the given axis by the specified angle.
+				[b]Note:[/b] The axis must be a normalized vector. Returns [code](0, 0, 0, 1)[/code] if your vector is not normalized in Debug mode. Returns the quaternion that would have been generated by normalizing the axis in release mode. If the axis has length [code]0[/code] and therefore cannot be normalized returns [code](0, 0, 0, 0)[/code]
```

Comment:
I do trust that this is all correct, as it is way out of my expertise. I **heavily** appreciate the extra detail.

However, the way this entire note and others are structured in this PR is extremely inconsistent and kind of "not good" for several reasons:
- `(0, 0, 0, 1)` is the **IDENTITY** Quaternion and should be referred to as such (`Returns [constant IDENTITY]`) to point users in the right direction, as it is not an arbitrary, magic value.
- "Debug" in "debug mode" is never capitalized across the entire class reference. Saying "debug builds" is also more common in this context.
- The whole note is extremely verbose. "Debug build" and "release build" are mutually exclusive, so there's almost no need to define both.
    - It also feels like the behavior in debug builds is the exception and not the norm, as the following sentence implies nothing wrong really happens when the axis is not normalized:
    _"Returns the quaternion that would have been generated by normalizing the axis in release mode."_
    This means that it's only worth noting the more "unusual" behavior in debug builds.
    - There's also little point in saying "_and therefore cannot be normalized_" because users interacting with a Quaternion should already have a decent understanding of vector math.

Reply:
I appreciate the feedback. I'll make those modifications. Great work on your documentation improvements by the way.